Tribler/Core/Upgrade/upgrade.py

```diff
@@ -55,7 +55,7 @@ def cleanup_noncompliant_channel_torrents(state_dir):
                     if name and len(name) != CHANNEL_DIR_NAME_LENGTH:
                         os.unlink(file_path)
                 except (TypeError, KeyError, ValueError):
-                    pass
+                    logger.debug("Malformed .pstate file %s", file_path)
```

Comment:
Could you make this logging statement a bit more specific? E.g.: "Malfored .pstate file %s found during cleanup of non-compliant channel torrents."

Reply:
Made it more descriptive per your feedback

Reply:
On a minor note, instead of hiding the message mentioning you I deleted it... Sorry about that...
For protocol its content was basically what I wrote above plus `@ name` of your name.